src/robots.txt.ejs

```diff
@@ -18,6 +18,10 @@ Disallow: /profile
 Disallow: /workflowitems
 # Crawlers should be able to access entity pages, but not the facet search links present on entity pages
 Disallow: /entities/*?f
+# do not crawl REST API endpoints and HAL browser
+Disallow: /server/*
```

Comment:
This assumes that the backend is deployed at `/server`.  It may not be deployed in that location.  That said, I think it's likely *many* sites will deploy the backend to `/server`.

We just may want to clarify that this path may require customization by adding something like:
```
# Do not crawl the REST API and HAL Browser
# NOTE: You may need to update this path if you deploy the REST API to a different location.
```

Reply:
Wouldn't that disallow the download of any bitstreams by search engines? For most repositories it is important that there contents (bitstreams themselves) are indexed by search engines.

Reply:
@pnbecker , maybe I'm missing something, but bitstream URLs in our DSpace instance have the form `/bitstreams/<uuid>/download`. Why do you see a conflict with the `/server` rule?

Reply:
yes, that checks if the file is available to you and forwards you to its actual url: `https://<url>/server/api/core/bitstreams/<uuid>/content`.

Reply:
Thank you, @pnbecker . I'll add an additional rule to `robots.txt` to allow crawler to harvest bitstreams.